Documentation/ceph-cluster-crd.md

```diff
@@ -824,19 +824,30 @@ The script will look for the following populated environment variables:
 
 * `NAMESPACE`: the namespace where the configmap and secrets should be injected
 * `ROOK_EXTERNAL_FSID`: the fsid of the external Ceph cluster, it can be retrieved via the `ceph fsid` command
-* `ROOK_EXTERNAL_ADMIN_SECRET`: the external Ceph cluster admin secret key, it can be retrieved via the `ceph auth get-key client.admin` command
 * `ROOK_EXTERNAL_CEPH_MON_DATA`: is a common-separated list of running monitors IP address along with their ports, e.g: `a=172.17.0.4:6789,b=172.17.0.5:6789,c=172.17.0.6:6789`. You don't need to specify all the monitors, you can simply pass one and the Operator will discover the rest. The name of the monitor is the name that appears in the `ceph status` output.
 
+Now, we need to give Rook a key to connect to the cluster in order to perform various operations such as health cluster check, CSI keys management etc...
```

Comment:
To document this feature what do you think about the following approach?
- When the key(s) are exported from the cluster, you always end up with five username/key pairs, one that is used by the operator and four that are used by the CSI driver.
- The script would always export the five keys with minimal access
- The script is just an example. They can modify the script to export keys with different username or privileges if desired. 
- Importing the keys will always expect to import the five keys. For backward compatibility it will allow only importing the admin key. However, I'd suggest we not document that case. To simplify the docs, we can just document the recommended approach of minimal access.